lib/common_ui/primary_container.dart

```diff
@@ -1,14 +1,21 @@
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
-import 'package:scouts_system/view_model/students.dart';
 
 // ignore: must_be_immutable
-class PrimaryContainerStudents extends StatelessWidget {
-  Students modelStudents;
+class PrimaryContainer extends StatelessWidget {
```

Comment:
Nice, but is it just a container? Because it seems to be much more than a container. Is it a "ListItem" or a "Card"?

Reply:
solved #14